modules/ui/src/app/pages/risk-assessment/profile-item/profile-item.component.html

```diff
@@ -14,17 +14,30 @@
 limitations under the License.
 -->
 <div class="profile-item-container">
-  <mat-icon class="profile-item-icon" fontSet="material-symbols-outlined">
-    rule
+  <mat-icon
+    *ngIf="profile.status === ProfileStatus.VALID"
+    class="profile-item-icon"
+    fontSet="material-symbols-outlined">
+    check_circle
   </mat-icon>
-  <p
-    class="profile-item-name"
-    role="button"
-    tabindex="0"
-    (click)="profileClicked.emit(profile)"
-    (keydown.enter)="profileClicked.emit(profile)">
-    {{ profile.name }}
-  </p>
+  <mat-icon
+    *ngIf="profile.status === ProfileStatus.DRAFT"
+    svgIcon="draft"
+    class="profile-draft-icon"></mat-icon>
+
+  <div class="profile-item-information">
+    <p
+      class="profile-item-name"
+      role="button"
+      tabindex="0"
+      (click)="profileClicked.emit(profile)"
+      (keydown.enter)="profileClicked.emit(profile)">
+      {{ profile.name }}
+    </p>
+    <p class="profile-item-created">
+      {{ profile.created | date: 'dd MMM yyyy' }}
```

Comment:
Based on the UI we also display this value on the download zip report. 
The link to the design: https://www.figma.com/design/C8U3CASvjyHV8Bf5S4ZMNz/Testrun-UI?node-id=4218-36310&m=dev
Should we also add it or there is another ticket?

Reply:
I think here, thanks